jvm/src/test/java/com/microsoft/bond/protocol/StreamHelperTest.java

```diff
@@ -0,0 +1,99 @@
+package com.microsoft.bond.protocol;
+
+import com.sun.xml.internal.ws.util.StreamUtils;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.EOFException;
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.junit.Assert.*;
+
+public class StreamHelperTest {
+
+    @Test
+    public void callImplicitDefaultConstructor() {
+        // adds the implicit default constructor to unit test code coverage
+        new StreamHelper();
```

Comment:
Fine, but can you give the helpers default-visibility nullary constructors or something in this change? It shouldn't be possible to construct a `StreamHelper`.